GitUI/UserControls/RevisionGrid/Graph/RevisionGraph.cs

```diff
@@ -414,7 +416,9 @@ private void BuildOrderedRowCache(RevisionGraphRevision[] orderedNodesCache, int
                 localOrderedRowCache.Add(new RevisionGraphRow(revision, segments));
             }
 
-            StraightenLanes(startIndex - _straightenLanesLookAhead, lastToCacheRowIndex, localOrderedRowCache);
+            int straightenStartIndex = Math.Max(1, startIndex - _straightenLanesLookAhead);
```

Comment:
why start with 1? (0?)

Reply:
> why start with 1?

The single node at index 0 shall never be moved.
Starting with 1, we always have a previous row to compare with.
As Igor suggested, I am adding a few comments.